### DISCUSSION TITLE
Two issues I’ve noticed

### DISCUSSION THREAD

**Discussion by spitfire2112** [isid0re/SoloLeveling#181] [Q&A] [answered]
I’ve noticed two separate issues while leveling two new characters.

The first is a pathing issue on a smiter paladin. I understand that once it finally rolls enigma that it’ll stop and the bot is mainly geared towards this. But it doesn’t seem to like running through mausoleum or cathedral style maps. If my guess is correct I believe it’s confusing itself with the multiple doorways / paths and just runs into walls between the two options.

The second issue I’ve noticed is while leveling a summon necro. I’ll check on the bot when I get home from work and a couple of times now I’ve seen it sitting in camp inside the stash. When I manually close the stash and allow it to go back to town it immediately summons the clay golem. I believe it’s attempting to summon the golem while still inside the stash after depositing items and gold.

I’m unsure if there is a fix for this but I thought I would bring it up to see if anyone else had seen or noticed these things as well as to bring it to your attention in case you were unaware so it could later be looked at when time allows.

I understand you have a lot of things on your plate but I also wanted to say thank you for the great stuff you are putting out. Keep it up!

Thanks

**Comment by isid0re** [comment, marked as answer]
I've been told about the pally problem in mausoleum, hadn't heard about the necro.  Thanks. I will look into the pather movements to see If there is something I can do.
